src/domain-registry-essentials/reference.md

```diff
@@ -1,6 +1,6 @@
 # Records Reference
 
-Records Reference is an overview of all standardized resolver record, that have a defined interpretation. Any other custom records are not forbidden to be set, however their interpretation remains custom to specific clients. See [Resolving Domain Records](resolving-domain-records.md) and [Architecture Overview Resolver](architecture-overview.md#resolver) for more information on domain records.
+Records Reference is an overview of all standardized resolver records that have a defined interpretation. Other custom records are allowed to be set. However, their interpretation remains custom to specific clients. See [Resolving Domain Records](resolving-domain-records.md) and [Architecture Overview Resolver](architecture-overview.md#resolver) for more information on domain records.
```

Comment:
> remains custom to specific clients
I'm thinking.. is "remains" a lexically correct choice here?

Reply:
Yes, I think... I tried to find a quick rule but there isn't one. `Remain` & `Remains` are two different words, and looking at the definition for both verb forms, either could be used properly here. Grammarly usually suggested changing to remains and IMO `remains` sounds a bit better